jep/0000/README.adoc

```diff
@@ -0,0 +1,298 @@
+= JEP-0000: Dispatchable Views and View Fragments in Stapler
+:toc: preamble
+:toclevels: 3
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+.**JEP Template**
+.Metadata
+[cols="1h,1"]
+|===
+| JEP
+| 0000
+
+| Title
+| Dispatchable Views and View Fragments in Stapler
+
+| Sponsor
+| https://github.com/jvz[Matt Sicker]
+
+// Use the script `set-jep-status <jep-number> <status>` to update the status.
+| Status
+| Not Submitted :information_source:
+
+| Type
+| Standards
+
+| Created
+| 2019-07-17
+
+| BDFL-Delegate
+| TBD
+
+//
+//
+// Uncomment if there is an associated placeholder JIRA issue.
```

Comment:
There is a Jira issue but it's hidden in the SECURITY project. Not sure of expected use in the case.